rcos_io/templates/projects/projects.html

```diff
@@ -6,7 +6,7 @@
 <div class="container">
 	<a href="/projects">Current Projects</a>
```

Comment:
Use `{{ url_for('projects.current_projects') }}` and `{{ url_for('projects.add_project') }}`

Reply:
Also, you can use `url_prefix='/projects'` like here https://github.com/Apexal/rcos_io/blob/815329136a982b2af401f1d201c848b683695976/rcos_io/auth.py#L24